css/main.scss

```diff
@@ -270,3 +270,7 @@ $on-laptop:        800px;
     }
 
 }
+
+.filter-body {
+    width: 100%;
+}
```

Comment:
```suggestion
```

Reply:
Move this rule into _sass/spectre/_filters.scss (see above)

Reply:
I think it should stay in main because the _sass/spectre is from the spectre framework and modifying it directly will complicate updating the framework

Reply:
reverted the corresponding commit